core/templates/css/oppia.css

```diff
@@ -696,30 +696,7 @@ textarea {
   margin-left: 4px;
   margin-top: 18px;
 }
-.oppia-sidebar-menu-open .oppia-sidebar-menu {
```

Comment:
These three classes -- why are they deleted (where are their replacements)? Or were they never used?

Reply:
At the time of creation, this PR also had the code for the migration of the side navigation bar, warnings-and-alerts component, and `ngx-toastr`. Later, I moved those migrations into respective individual PRs. By mistake, I forgot to revert changes to this file. I have fixed this now and removed all the changes which seemed unnecessary to me.